specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2021-01-15/cosmos-db.json

```diff
@@ -8311,7 +8311,6 @@
     },
     "PublicNetworkAccess": {
       "type": "string",
-      "readOnly": true,
       "description": "Whether requests from Public Network are allowed",
```

Comment:
We need to update examples too.

Reply:
There is no examples using  PublicNetworkAccess property to modify. This is only used in Database Account related operations.

Reply:
DatabaseAccount-create max example would not need it ?

Reply:
If this is not a readonly property then it can be updated/created then the above mentioned example should be modified. Let me know if you think otherwise

Reply:
Got it. Done.